tests/test_schema.py

```diff
@@ -1837,7 +1837,10 @@ class Model(BaseModel):
 def test_frozen_set():
     class Model(BaseModel):
         a: FrozenSet[int] = frozenset({1, 2, 3})
+        b: FrozenSet = frozenset({1, 2, 3})
+        c: frozenset = frozenset({1, 2, 3})
```

Comment:
```suggestion
        c: frozenset = frozenset({1, 2, 3})
        d: frozenset = ...
```
Just to check the case when there's no default (obviously you'll need to change the check below too.